### PR TITLE
Fix name and icon for notification

### PR DESCRIPTION
**Pull request by GeekCornerGH**
### Description

So, when you will receive a toaster notification, you will see Remote.it app name and icon,

### Checklist

- [ x ] This PR only contains one isolated change (bug fix, feature, etc) (yes, as far I know)
- [ x ] I have manually reviewed this PR to make sure only the files I intended to change were changed and nothing else

### Test plan

1. Go on a Windows 10 computer
2. Put a shortcut of Electron.exe (in `node_modules\electron\dist\electron.exe`) in your start menu, then give him as name: "Remote.it" and as logo a Remote.it Logo
3. Start the app then open Devtools
4. Type
```js
new Notification("Test")
```
5. You should see as Notification name "Remote.it" and as logo, the logo of Remote.it

Thank you for making that awesome software :)

### CODE REVIEW

File: src/ElectronApp.ts [hunk 1]
```diff
@@ -122,6 +122,7 @@ export default class ElectronApp {
   private createMainWindow = () => {
     d('Create main window')
     if (this.window) return
+    this.app.setAppUserModelId(process.execPath)
```
Comment (mauricio0000) [inline src/ElectronApp.ts:125]: @JamieRuderman it is working

File: src/ElectronApp.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default class ElectronApp {
     this.autoUpdater = new AutoUpdater()
     this.protocol = process.env.NODE_ENV === 'development' ? DEEP_LINK_PROTOCOL_DEV : DEEP_LINK_PROTOCOL
 
-    if (!this.app.requestSingleInstanceLock()) {
```
Comment (mauricio0000) [inline src/ElectronApp.ts:33]: this one is broken the build

Comment (GeekCornerGH) [reply src/ElectronApp.ts:33]: Hmm, I never added this, its maybe from your side?

Comment (GeekCornerGH) [reply src/ElectronApp.ts:33]: Ohh i see sorry

**Review by mauricio0000 — APPROVED** [review]

**Pull request merged**